packages/components/src/search/autocompleters/taxes.js

```diff
@@ -12,8 +12,7 @@ import { stringifyQuery } from '@woocommerce/navigation';
 /**
  * Internal dependencies
  */
-import { computeSuggestionMatch } from './utils';
-import { getTaxCode } from 'analytics/report/taxes/utils';
```

Comment:
You can delete this file now -> `analytics/report/taxes/utils`

Reply:
It is still used here:

```
analytics/report/taxes/config.js
11:import { getTaxCode } from './utils';
54:						label: getTaxCode( tax ),
```

Reply:
Oh I see 🚢